CloneTiving/CloneTiving/ViewControllers/Main/Home/MainViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  MainViewController.swift
+//  CloneTiving
+//
+//  Created by 지희의 MAC on 2023/05/02.
+//
+
+import UIKit
+
+class MainViewController: BaseViewController {
+    
+    //MARK: UI
+    
+    
+    private let categoryTitleList = [ "홈", "실시간", "TV프로그램", "영화", "파라마운트+", "키즈" ]
+    
+    lazy var pageViewController : UIPageViewController = {
+        let pageVC = UIPageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal)
+        return pageVC
+    }()
```

Comment:
p2;
```suggestion
    lazy var pageViewController  = UIPageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal)
```
요롷게 줄일 수 있을듯요!